lib/active_job/job_proxy.rb

```diff
@@ -23,4 +23,10 @@ def serialize
   def perform_now
     raise UnsupportedError, "A JobProxy doesn't support immediate execution, only enqueuing."
   end
+
+  ActiveJob::JobsRelation::STATUSES.each do |status|
+    define_method "#{status}?" do
+      self.status == status
+    end
+  end
```

Comment:
See comment https://github.com/basecamp/mission_control-jobs/pull/74/files#r1505001794. If we keep this, the `status` method in  `ActiveJob::Base` (via `ActiveJob::Executing`) should go.

Reply:
Yes, we need keep this to support `blocked?` method and etc.